docs/cmd/kn_domain_describe.md

```diff
@@ -21,6 +21,7 @@ kn domain describe NAME
   -h, --help                          help for describe
   -n, --namespace string              Specify the namespace to operate in.
   -o, --output string                 Output format. One of: json|yaml|name|go-template|go-template-file|template|templatefile|jsonpath|jsonpath-as-json|jsonpath-file|url.
+      --show-managed-fields           If true, keep the managedFields when printing objects in JSON or YAML format.
```

Comment:
Does this work ? So are managed fields hidden by default now ?

Reply:
Yes, it does. The change is part of updated `k8s.io/cli-runtime v0.21.4`. The latest `kubectl` hides managed fields as well.

E.g.:
```
➜  client git:(pr/cleanup-replace) kn service describe hello -o yaml --show-managed-fields
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  annotations:
    serving.knative.dev/creator: kubernetes-admin
    serving.knative.dev/lastModifier: kubernetes-admin
  creationTimestamp: "2021-09-02T10:53:19Z"
  generation: 1
  managedFields:
  - apiVersion: serving.knative.dev/v1
    fieldsType: FieldsV1
    fieldsV1:
      f:spec:
        .: {}
        f:template:
          .: {}
          f:metadata:
            .: {}
            f:annotations:
              .: {}
              f:client.knative.dev/updateTimestamp: {}
              f:client.knative.dev/user-image: {}
            f:creationTimestamp: {}
          f:spec:
            .: {}
            f:containers: {}

```
vs.
```
➜  client git:(pr/cleanup-replace) kn service describe hello -o yaml                      
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  annotations:
    serving.knative.dev/creator: kubernetes-admin
    serving.knative.dev/lastModifier: kubernetes-admin
  creationTimestamp: "2021-09-02T10:53:19Z"
  generation: 1
  name: hello
  namespace: default
  resourceVersion: "222953"
  uid: 95036efa-c9f9-4065-b1c0-f7794d0b1ad0
```